components/movementsensor/adxl345/adxl345.go

```diff
@@ -67,6 +68,7 @@ type TapConfig struct {
 	ExcludeZ         bool    `json:"exclude_z,omitempty"`
 	Threshold        float32 `json:"threshold,omitempty"`
 	Dur              float32 `json:"dur_us,omitempty"`
+	DebounceMs       int     `json:"debounce_ms,omitempty"`
```

Comment:
let's call this `tap_debounce_ms`

Reply:
I like Rand's name! I can imagine adding other debounces in the future; this one is specifically for tap detection.

Reply:
This is in the tapConfig so I felt like including tap in the name is redundant

Reply:
My mistake! You're right that if it's already in the tap config, we shouldn't have redundant naming.